lib/core/utils/navigation_utils.dart

```diff
@@ -1,32 +1,39 @@
 import 'package:app/core/data/notification/notification_constants.dart';
 import 'package:app/core/domain/notification/entities/notification.dart' as NotificationEntities;
 import 'package:app/core/utils/dialog_utils.dart';
+import 'package:app/router/app_router.gr.dart';
+import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
 
 class NavigationUtils {
 
-  static handleNotificationNavigate(BuildContext context, NotificationEntities.Notification? notification) {
-    switch(notification?.object_type) {
+  static handleNotificationNavigate(BuildContext context, String? type, String? objectType, String? objectId) {
+    switch(objectType) {
       case "StoreOrder": 
```

Comment:
Please make use of enum 
ex: 
enum NotificationType {
StoreOrder,
....
}

Reply:
Resolved